src/components/fruitCounter/fruitCounter.jsx

```diff
@@ -0,0 +1,14 @@
+import "./fruitCounter.css"
+
+function FruitCounter({ fruit, emoji, count, updateCount }) {
+    return (
+        <article>
+            <h3>{emoji} {fruit}</h3>
+            <button type="button" onClick={() => updateCount(fruit, -1)}>-</button>
```

Comment:
Je hebt een Button Component gemaakt echter gebruik je die hier niet..

Je had ook hier disabled kunnen gebruiken om een knop te disabelen als het getal 0 is zodat je niet in de min kan.